Berksfile.lock

```diff
@@ -4,6 +4,8 @@ DEPENDENCIES
     metadata: true
 
 GRAPH
-  netuitive (0.17.0)
+  apt (3.0.0)
+  netuitive (0.19.0)
+    apt (>= 0.0.0)
```

Comment:
This merge wasn't done correctly; we no longer depend on the `apt` cookbook because modern Chef doesn't